external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -110,6 +110,34 @@ private[kafka010] class KafkaOffsetReader(
     kafkaReaderThread.shutdown()
   }
 
+  /**
+   * Commits offsets to the consumer_offsets topic
+   * for the driver consumer.
+  */
+  def commitOffsetsAsync(
+      partitionToOffsets: Map[TopicPartition, Long],
+      s: KafkaCommitsSource)
```

Comment:
I'd rather at least use `src` even we want to have short name - `s` doesn't say anything.